### ISSUE TITLE
[Chore]: Speed up e2e tests 

### ISSUE THREAD

**Issue by jsjoeio** [coder/code-server#5435]
## What is your suggestion?

To speed up our e2e tests using one of the two options:

### Option 1 - one test per file

> By default, tests in a single file are run in order. If you have many independent tests in a single file, you might want to run them in parallel with [test.describe.configure([options])](https://playwright.dev/docs/api/class-test#test-describe-configure).

They won't parallelize multiple tests in a single file so we could split tests out to individual files.

[Source](https://playwright.dev/docs/test-parallel#parallelize-tests-in-a-single-file)

### Option 2 - figure out how to use configure

Supposedly there is a method: `test.describe.configure({ mode: 'parallel' });`. However, we extend `test` and it doesn't seem to exist when I try to use it locally. I've only managed to get `test.describe.parallel` semi-working.

## Why do you want this feature?

I want to speed up CI and cut down on e2e run times. 

<img width="344" alt="image" src="https://user-images.githubusercontent.com/3806031/183773059-cc710da4-edbf-4590-b7f4-1092ddc9427b.png">

## Are there any workarounds to get this functionality today?

No

## Are you interested in submitting a PR for this?

Yes

**Comment by jsjoeio** [comment]
Based on the research I've done so far, it seems like we have a couple things we can do:
- increase workers to 2-4
- decrease timeout (increase for long-running individual tests)

We also want to try the individual vs many files with more workers and see if that helps.

**Comment by jsjoeio** [comment]
## Results

After running the tests with a different number of workers and the tests files split grouped vs individual, here are the results (based on 1 passing run in CI:

| Workers |  | Grouped | Individual |
|:---:|---|---|---|
| 1 |  | tbd s | tbd s |
| 2 |  | tbd s | tbd s |
| 4 |  | tbd s | tbd s |

**Comment by jsjoeio** [comment]
## Next Steps

- [ ] Close PRs
- [ ] Open new PR that adjusts timeouts
- [ ] Track fixing flakey tests [here](https://github.com/coder/code-server/issues/5482)

**Comment by jsjoeio** [comment]
I think the best move is to actually do https://github.com/coder/code-server/issues/5482 instead. That will speed up tests.
